contributing.md

```diff
@@ -0,0 +1,38 @@
+# How To Become A Contributor And Submit Your Own Code
+There are few guidelines that we need contributors to follow so that we can have a chance of keeping on top of things.
+
+## Getting Started
+This repo uses Lerna(https://lernajs.io/) as a top level setup, which is a tool that optimizes the workflow around managing multi-package repositories with git and npm.
+
+* Make sure you have forked the desired repo(https://github.com/electrode-io/electrode.git) for developing and testing your code changes.
+
+* Make sure you have installed Lerna and gulp-cli
+
+````
+$ npm install -g lerna gulp-cli
+````
+
+* Run `lerna bootstrap` for bootstrapping the packages in the current Lerna repo. It will install all their dependencies and linking any cross-dependencies.
```

Comment:
before running `npm run bootstrap`, needs to do `npm install` first.
So anyways, full steps:
```
npm install -g gulp-cli
git clone https://github.com/electrode-io/electrode.git
cd electrode
npm install
npm run bootstrap
gulp samples-local
cd samples/universal-react-node
npm install
gulp dev|hot
```